pyspecProcScripts/integral_w_error.py

```diff
@@ -43,23 +44,25 @@ def integral_w_errors(s,sig_path,error_path, indirect='vd', direct='t2',fl=None,
              +str(extra_dims))
     collected_variance = ndshape(
          [ndshape(s)[indirect],len(error_path)],[indirect,'pathways']).alloc()
+    # this is not averaging over all the pathways!!!!
     for j in range(len(error_path)):
-     # calculate N₂ Δf² σ², which is the variance of the integral (by error propagation)
-     # where N₂ is the number of points in the indirect dimension
-     s_forerror = select_pathway(s,error_path[j])
-     if j==0: N2 = ndshape(s_forerror)[direct]
-     # mean divides by N₁ (indirect), integrate multiplies by Δf, and the
-     # mean sums all elements (there are N₁N₂ elements)
-     s_forerror -= s_forerror.C.mean_all_but([indirect, direct]).mean(direct)
-     s_forerror.run(lambda x: abs(x)**2/2).mean_all_but([direct,indirect]).mean(direct)
-     s_forerror *= df**2 # Δf
-     s_forerror *= N2
+        # calculate N₂ Δf² σ², which is the variance of the integral (by error propagation)
+        # where N₂ is the number of points in the indirect dimension
+        s_forerror = select_pathway(s,error_path[j])
```

Comment:
notice how this wipes everything out and starts over!!!!

this is part of the the reason for #44 

note that this came from some version where I had generated `collected_error` above -- for some reason, you got rid of that and did this instead.

Reply:
added to it, if I remember right